cmd/vsphere-cloud-controller-manager/main.go

```diff
@@ -80,7 +81,7 @@ func main() {
 	}
 
 	fs := command.Flags()
-	namedFlagSets := ccmOptions.Flags(app.ControllerNames(app.DefaultInitFuncConstructors), app.ControllersDisabledByDefault.List(), app.AllWebhooks, app.DisabledByDefaultWebhooks)
+	namedFlagSets := ccmOptions.Flags(app.ControllerNames(app.DefaultInitFuncConstructors), app.ControllersDisabledByDefault.List(), names.CCMControllerAliases(), app.AllWebhooks, app.DisabledByDefaultWebhooks)
```

Comment:
New KCM and CCM change on 1.28: https://github.com/kubernetes/kubernetes/pull/115813